test/curve_analysis/test_baseclass.py

```diff
@@ -226,12 +231,43 @@ def test_end_to_end_single_function(self):
         result = analysis.run(test_data)
         self.assertExperimentDone(result)
 
+        curve_data = result.artifacts("curve_data").data
+        np.testing.assert_array_equal(curve_data.series_name, "test")
+        np.testing.assert_array_equal(curve_data.analysis, "CurveAnalysis")
+        np.testing.assert_array_equal(len(curve_data.filter(category="raw")), 100)
```

Comment:
```suggestion
        self.assertEqual(len(curve_data.filter(category="raw")), 100)
```
Why testing with assert_array_equal?

Reply:
Oops, copy paste error, thanks.